src/instrumentation.py

```diff
@@ -0,0 +1,124 @@
+from prometheus_client import CollectorRegistry, Counter, Gauge
+
+from src.constants import TASK_EVENT_LABELS, WORKER_EVENT_LABELS, LabelName
+
+registry = CollectorRegistry(auto_describe=True)
+
+
+class EventInstrumentMixin:
```

Comment:
I seriously do not get it why `prometheus_client` project does not expose those values natively using read-only properties but now we have it :).